### PR TITLE
[MNOE-143] - Ability to disable adding new app

### PR DESCRIPTION
**Pull request by fgourichon**
@ouranos and @alexnoox for review

### CODE REVIEW

File: src/app/components/dashboard-apps-dock/dashboard-apps-dock.coffee [hunk 1]
```diff
@@ -108,6 +110,9 @@ DashboardAppsDockCtrl = ($scope, $cookies, $uibModal, $window, MnoeOrganizations
           $scope.apps = MnoeAppInstances.appInstances
       )
 
+  $scope.displayDock = ->
```
Comment (ouranos) [inline src/app/components/dashboard-apps-dock/dashboard-apps-dock.coffee]: Add a comment explaining why you hide the dock?
`# Hide the dock when no apps and marketplace is disabled`

File: src/app/index.default-config.coffee [hunk 2]
```diff
@@ -16,3 +16,4 @@ angular.module('mnoEnterprise.defaultConfiguration', [])
   .constant('GOOGLE_TAG_CONTAINER_ID', null)
   .constant('APP_NAME', null)
   .constant('INTERCOM_ID', null)
+  .constant('MARKETPLACE_CONFIG', {enabled: true})
```
Comment (ouranos) [inline src/app/index.default-config.coffee]: Merge it with L15

File: src/app/index.route.coffee [hunk 3]
```diff
@@ -37,6 +37,21 @@ angular.module 'mnoEnterpriseAngular'
         templateUrl: 'app/views/company/company.html'
         controller: 'DashboardCompanyCtrl'
         controllerAs: 'vm'
+      .state 'logout',
```
Comment (ouranos) [inline src/app/index.route.coffee]: Did you just swap the routes here? I can't see where `MARKETPLACE_CONFIG` is used?

Comment (fgourichon) [inline src/app/index.route.coffee]: Bad commit...

**Comment by fgourichon** [comment]
@ouranos, changes done

File: src/app/components/dashboard-apps-dock/dashboard-apps-dock.coffee [hunk 4]
```diff
@@ -108,6 +110,10 @@ DashboardAppsDockCtrl = ($scope, $cookies, $uibModal, $window, MnoeOrganizations
           $scope.apps = MnoeAppInstances.appInstances
       )
 
+  # Hide the dock if marketplace is disabled and there is not app linked
+  $scope.displayDock = ->
+    return if !$scope.isMarketplaceEnabled && (!$scope.apps? || $scope.apps.length == 0 ) then false else true
```
Comment (ouranos) [inline src/app/components/dashboard-apps-dock/dashboard-apps-dock.coffee]: Oh I missed this one

```coffescript
$scope.displayDock = ->  
  $scope.isMarketplaceEnabled || ($scope.apps? && $scope.apps.length > 0)
```

* avoid `if condition then true else false`
* return are implicit in coffescript

Comment (fgourichon) [inline src/app/components/dashboard-apps-dock/dashboard-apps-dock.coffee]: So obvious...

**Pull request merged**